### PR TITLE
metrics: fix wrong metrics for balance region

### LINKED ISSUE #6970: metrics: `balance region scheduler` metrics not work
### What did you expect to see?
scheduler work well
### What did you see instead?
image

### What version of PD are you using (`pd-server -V`)?
master

### PR DESCRIPTION
**Pull request by bufferflies**
### What problem does this PR solve?

Issue Number: Close  #6970

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Manual test (add detailed scripts or steps below)
image

image

image

Code changes

Side effects

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by JmPotato — APPROVED** [review]

**Comment by HuSharp** [comment]
please add an issue to describe the problem

File: pkg/schedule/schedulers/metrics.go [hunk 1]
```diff
@@ -33,14 +33,6 @@ var (
 			Help:      "Counter of scheduler events.",
 		}, []string{"type", "name"})
 
-	schedulerStatus = prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Namespace: "pd",
-			Subsystem: "scheduler",
-			Name:      "inner_status",
```
[the remark this answers is not in the document]
Comment (bufferflies) [reply pkg/schedule/schedulers/metrics.go:40]: no need, the only used is pending_op_failed.

**Comment by HuSharp** [comment]
please paste `balance-region-movement` and `balance-leader-movement` as well.

**Comment by bufferflies** [comment]
> please paste `balance-region-movement` and `balance-leader-movement` as well.

fixed

**Review by HuSharp — APPROVED** [review]

**Pull request merged**